porting/common-kernel-build-errors.rst

```diff
@@ -51,6 +47,41 @@ Example of the error::
    ../../../../../../kernel/lenovo/msm8916/fs/ecryptfs/file.c:130:16: error: assignment of read-only member 'actor'
   buf.ctx.actor = ecryptfs_filldir;
 
-Apply `this patch from bullhead`_.
+Apply `'patch ecryptfs to fix a build error' from bullhead`_.
+
+'Undefined reference to pidns_operations' on Linux 3.4
+------------------------------------------------------
+
+The implementation of PID Namespacing was incomplete in the Android kernel 3.4, causing the following error::
+
+   fs/built-in.o:namespaces.c:ns_entries: error: undefined reference to 'pidns_operations'
+
+To fix this issue, apply the patch `Finish implementation of PID namespace`_
+
```

Comment:
oh, actually @UniversalSuperBox I have this error at this point https://github.com/doniks/android_kernel_google_msm/commit/cf47294ba99cbb11515521ba7dffb99c93a48d3f 
with this patch, it compiles, but then at runtime I get this error: http://paste.ubuntu.com/p/9f7N8dDwWs/ however if I alternatively use these two: https://github.com/doniks/android_kernel_google_msm/commit/4d943958ff9e788f6ea7fe745c191137047eb9b8 https://github.com/doniks/android_kernel_google_msm/commit/0525c54b9add624829df432d613a1c4cbb0d0b97 I dont get the error, but get up to UT booting and apps running (with some more patches). So, I'm not sure whether we universally want to suggest this patch. 

What do you think?

Reply:
@doniks: We don't have issues with 3.4 kernels on Mako, Hammerhead, Onyx, Tenderloin. We use for example https://github.com/Halium/android_kernel_lge_hammerhead/commit/bd221854de33b75db7a7fa01cb34274b62a7cbf8

Reply:
that makes it five kernels against one :) ok, lets put it in